src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -0,0 +1,607 @@
+import logging
+import math
+import os
+import threading
+import time
+
+import numpy
+import psutil
+from concurrent.futures._base import RUNNING, FINISHED, CANCELLED, CancelledError
+
+from odemis import model, dataio
+from odemis.acq import acqmng
+from odemis.acq import stitching
+from odemis.acq.align.autofocus import MeasureOpticalFocus, AutoFocus, MTD_EXHAUSTIVE
+from odemis.acq.stream import Stream, SEMStream, CameraStream, RepetitionStream, EMStream, ARStream, \
+    SpectrumStream, FluoStream, MultipleDetectorStream, util, executeAsyncTask, \
+    CLStream
+from odemis.util import dataio as udataio
+from odemis.util.comp import compute_scanner_fov, compute_camera_fov
+
+WEAVER_COLLAGE_REVERSE = 2
+# TODO: Find a value that works fine with cryo-secom
+# Percentage of the allowed difference of tile focus from good focus
```

Comment:
That's not a "percentage", but a "ratio". I'd also add "The bigger, the less likely the autofocus will be triggered".